### ISSUE TITLE
Wand ceased to exist

### ISSUE THREAD

**Issue by ze-miner3000** [Wynncraft/Issues#5608]
My wand, Sage, ceased to exist while I was in the Olux swampland going to do a quest. It wasn't very good though (10% XP bonus) but I have no idea where it went, as I didn't drop it.

**Comment by ze-miner3000** [comment]
I currently have no usable wands.

**Comment by ze-miner3000** [comment]
nvm my friend is giving me his Sage :D

**Comment by Trixter313** [comment]
It was model likely dropped without you noticing. We reccomend changing your drop key to something you're less likely to accidentally press to prevent this from happening.

**Comment by ze-miner3000** [comment]
That seems unlikely, as my drop key is bound to R and I would have noticed it if I dropped it. I did however open a loot chest nearby, and I know that whatever is inside of a loot chest when it is closed is deleted forever. I would suggest tweaking the game so that whatever is inside of a loot chest when it disappears does not get deleted if you put it in there, but would have a very fast despawn timer of about 10 seconds.
